source/Government.cpp

```diff
@@ -93,8 +95,6 @@ void Government::Load(const DataNode &node)
 				hostileDisabledHail = nullptr;
 			else if(key == "language")
 				language.clear();
-			else if(key == "enforces")
-				enforcementZones.clear();
```

Comment:
What did this used to be?

Reply:
A merge error, good catch, and sorry ^^'